src/main/resources/assets/api-docs.yaml

```diff
@@ -1863,6 +1863,19 @@ paths:
             type: array
             items:
               type: string
+  /api/dataset/test:
+    post:
+      summary: Creates the Dataset from JSON
+      description: Creates the Dataset from JSON (WIP)
+      consumes:
+        - application/json
+      tags:
+        - Datasets
+      responses:
+        200:
+          description: Successfully created Dataset
+          schema:
+            $ref: '#/definitions/Dataset'
```

Comment:
This should be a 201 created response. 
Also, we'll need to document a 400 here for when the JSON is not valid, and we'll also need a 500 for any other error.

Reply:
As an aside, MDN is generally a good source of info for web development. They have an organized list of http status codes here.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

I've always kept it bookmarked as a good reference